.github/workflows/scripts/e2e.container.default.verify.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# shellcheck source=/dev/null
+source "./.github/workflows/scripts/e2e-verify.common.sh"
+
+go env -w GOFLAGS=-mod=mod
+
+# verify_provenance_content verifies provenance content generated by the container generator.
+verify_provenance_content() {
+    ATTESTATION=$(jq -r '.payload' <"$PROVENANCE" | base64 -d)
+
+    echo "  **** Provenance content verification *****"
+
+    # Verify all common provenance fields.
+    e2e_verify_common_all "$ATTESTATION"
+
+    e2e_verify_predicate_subject_name "$ATTESTATION" "$CONTAINER"
+    e2e_verify_predicate_builder_id "$ATTESTATION" "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/heads/main"
+    e2e_verify_predicate_buildType "$ATTESTATION" "https://github.com/slsa-framework/slsa-github-generator/generic@v1"
+}
+
+THIS_FILE=$(e2e_this_file)
+BRANCH=$(echo "$THIS_FILE" | cut -d '.' -f4)
+echo "branch is $BRANCH"
+echo "GITHUB_REF_NAME: $GITHUB_REF_NAME"
+echo "GITHUB_REF_TYPE: $GITHUB_REF_TYPE"
+echo "GITHUB_REF: $GITHUB_REF"
+echo "DEBUG: file is $THIS_FILE"
+
+# Verify provenance authenticity.
+e2e_run_verifier_all_releases "HEAD"
+
+# Verify provenance content.
+verify_provenance_content
+
+
```

Comment:
nit: do we need these extra lines?

Reply:
no! surprised shellcheck didn't catch this

Reply:
I don't think shellcheck has a check for this? (at least not one that's on by default)

I usually just run `shfmt` at it clears this up for me automatically.